packages/theme/src/plugins/i18n.ts

```diff
@@ -0,0 +1,21 @@
+import { createI18n } from "vue-i18n";
+
+//locales
+import en from "../locales/en.json";
+import fr from "../locales/fr.json";
+import hi from "../locales/hi.json";
```

Comment:
@Vedant005 can you also check if we can split the translations.

Like moving `ee` relates translations to `packages/theme/src/ee/locales` directory.